.gitlab-ci.yml

```diff
@@ -0,0 +1,151 @@
+---
+# To contribute improvements to CI/CD templates, please follow the Development guide at:
+# https://docs.gitlab.com/ee/development/cicd/templates.html
+stages:
+  - test
+  - build
+  - build_gdcdatamodel2
+  - release_gdcdatamodel2
+
+include:
+  - project: nci-gdc/gitlab-templates
+    ref: master
+    file:
+      - templates/global/full.yaml
+      - templates/python/full.yaml
+
+variables:
+ BASE_CONTAINER_VERSION: "1.4.0"
+ PRE_COMMIT_DOCKER_IMAGE: docker.osdc.io/ncigdc/python3.8-builder:2.1.0
```

Comment:
I think our pre-commit version needs to be 3.7 if we continue with the idea that we support the oldest supported version of python.